Lib/test/test_urllib2.py

```diff
@@ -808,12 +808,12 @@ def test_file(self):
             self.assertEqual(respurl, url)
 
         for url in [
-            "file://localhost:80%s" % urlpath,
+            "file://localhost:80/%s" % urlpath,
             "file:///file_does_not_exist.txt",
             "file://not-a-local-host.com//dir/file.txt",
-            "file://%s:80%s/%s" % (socket.gethostbyname('localhost'),
+            "file://%s:80/%s/%s" % (socket.gethostbyname('localhost'),
```

Comment:
Wouldn't this and the next change have doubled slashes because `getcwd()` returns a leading slash?

Reply:
This was breaking tests on windows because the test data included a dir path.